src/OpenTelemetry.Instrumentation.Http/README.md

```diff
@@ -68,7 +68,7 @@ public class Program
 #### Metrics
 
 > **Note**
-> Metrics are not available for .NET Framework.
+> Metrics are only available for .NET Framework when tracing is enabled.
```

Comment:
I think we need to add more info here. Its not just sufficient to enable Tracing, but the sampler should also be returning Recorded+Sampled for each span in-order for metrics to work. As Metrics rely on Activity to populate tags, and Activity itself won't be populated unless Sampler decision is favorable.

Reply:
@cijothomas please check my latest revision, better clarified what is needed to enable tracing